test/subspace-test-runtime/src/lib.rs

```diff
@@ -119,11 +119,11 @@ pub const SSC: Balance = (10 * SHANNON).pow(DECIMAL_PLACES as u32);
 ///
 /// Based on:
 /// <https://research.web3.foundation/en/latest/polkadot/block-production/Babe.html#-6.-practical-results>
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
+pub const MILLISECS_PER_BLOCK: u64 = 12000;
 
 // NOTE: Currently it is not possible to change the slot duration after the chain has started.
 //       Attempting to do so will brick block production.
-const SLOT_DURATION: u64 = 1000;
+const SLOT_DURATION: u64 = 2000;
```

Comment:
These will make tests very slow.

Reply:
The block production would fail with the normal setting because it would take too long, and even the block time has been doubled, you still need to run the tests in release mode, it still failed if you run it in the debug mode.

Reply:
Why would it take too long?

Reply:
Because we're still using the slot based mechanism and the machine is not capable of running multiple nodes simultaneously and still being able to produce blocks in a pretty limited time. https://github.com/paritytech/substrate/blob/4cd2cc958f5a078d87e4eae877167c40cf2ebf9e/client/consensus/slots/src/lib.rs#L318

Reply:
Why not capable? Node takes a tiny fraction of CPU core to run, you should be able to run 10 nodes on a single core, especially when blockchain starts. Even in debug mode it should be feasible to have many nodes running concurrently on the same core. If not we're doing something wrong. We can't run tests that require many blocks with 0.5 blocks per second speed, it will take forever in CI and even locally. In release mode I was running node with 10 blocks/s speed without issues.

Reply:
Well, the argument makes sense, but I don't see anything obviously wrong. I can dig into that if you want, just not sure how much time it will take.

Reply:
Please do dig into it a little bit, it shouldn't take anywhere close to a second to produce a block.

Reply:
Turned out that `runtime_cache_size` was misconfigured, fixed in https://github.com/subspace/subspace/pull/266/commits/89795e18cd05993d3f38b23a160bc54cb1f401e9